client/common/cmdline.c

```diff
@@ -410,11 +410,28 @@ static size_t print_description(const char* text, size_t start_offset, size_t cu
 	return current;
 }
 
-static void freerdp_client_print_command_line_args(const COMMAND_LINE_ARGUMENT_A* arg)
+static int cmp_cmdline_args(const void* pva, const void* pvb)
 {
-	if (!arg)
+	const COMMAND_LINE_ARGUMENT_A* a = (const COMMAND_LINE_ARGUMENT_A*)pva;
+	const COMMAND_LINE_ARGUMENT_A* b = (const COMMAND_LINE_ARGUMENT_A*)pvb;
+
+	if (!a->Name && !b->Name)
```

Comment:
Shouldn't this actually be an `assert(a->Name && b->Name)` (and the following name checks removed)?
Can't think of any reason this could be a valid case